lib/onebox/engine/twitter_status_onebox.rb

```diff
@@ -39,66 +71,139 @@ def twitter_api_credentials_present?
         client && !client.twitter_credentials_missing?
       end
 
-      def raw
-        @raw ||= client.status(match[:id]).to_hash if twitter_api_credentials_present?
+      def symbolize_keys(obj)
```

Comment:
There's a `symbolize_keys` method in `onebox/helpers.rb`. Any chance that can be reused here? (I haven't looked into the details.)

Reply:
I missed that one. But I tested it, and it doesn't handle nested objects in arrays